v5/pkg/walkers/sql/walk.go

```diff
@@ -177,6 +177,13 @@ func Walk(n tsl.Node) (s sq.Sqlizer, err error) {
 		s = sq.Expr(f)
 	case tsl.StringOp:
 		s = sq.Expr(n.Left.(string))
+	case tsl.BooleanOp:
+		// No Booleans in SQL, convert to 0 for false and 1 for true.
```

Comment:
This is not true.  SQL standard defines a boolean type, with `TRUE`/ `FALSE`/`UNKNOWN` literal syntax.
https://crate.io/docs/sql-99/en/latest/chapters/09.html

_Of course_, not all databases implement even such simple parts of the standard...   And being a vendors-driven committee, the standard is careful to let vendors claim partial compliance without it :man_facepalming:

For example MySQL has no real distinct bool type, just an alias to a 1-byte int.  It does parse `TRUE` and `FALSE` literals as 1 and 0.  (I'm not sure whether that counts as standard-compliant.)
https://dev.mysql.com/doc/refman/5.6/en/boolean-literals.html

PostgreSQL has a real boolean type.  It does accept 1 and 0 as alternative literals, so this will work there too I believe.  
But strictly speaking, a full implementation of the standard doesn't have to accept 1/0 in bool comparisons, so emitting `TRUE` / `FALSE` would be best IMHO.